liby2/src/Y2SerialComponent.cc

```diff
@@ -344,23 +344,25 @@ bool Y2SerialComponent::initializeConnection()
 
 	 // first write one space so the other side gets
 	 // fullfilled its necessities
-	 write(fd_serial, space_buf, 1);
+	 (void) (write(fd_serial, space_buf, 1) == 1);
```

Comment:
to be honest this `(void)` stuff is similar "nasty" as blank `if` which I also do not like, as it just add noise to code.